### DISCUSSION TITLE
百宝箱代码去哪了?

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [Meloong-Git/PCL#1340] [提问] [answered]
想去刨下载模块的代码 orz

**Comment by WForst-Breeze** [comment, marked as answer]
百宝箱代码尚未公布，请见 #455

以下是本仓库未公布的代码：
> 包含主题解锁、加密解密等保密代码
ModSecret.vb  
设置存取系统，这包含了部分玩家账号信息
ModSetup.vb  
包含主题解锁代码（关于页面、个性化设置页面）
PageOtherAbout.*
PageSetupUI.*  
包含回声洞、开发者测试工具（百宝箱页面）
PageOtherTest.*  
开发者工具
ModDevelop.vb  
不公开底层函数库，以免抄袭
ModMain.vb

**Comment by Chiloven945** [comment]
你这算盘可以在火星听见
